sale_order_type/__manifest__.py

```diff
@@ -19,8 +19,7 @@
     "website": "https://github.com/OCA/sale-workflow",
     "license": "AGPL-3",
     "depends": [
-        'sale_stock',
-        'account',
+        'sale_stock'
```

Comment:
why did you remove `account` from `depends`?

Reply:
@tafaRU because it is redundant: it is already included because `sale_stock` depends on `sale` that depends on `account`